src/components/Pokedex/PokemonEggGroups.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Container, Typography } from '@mui/material';
+import { getEggGroupNameById } from '../../utils/dex/egggroup';
+
+export const PokemonEggGroups = ({ eggGroupIds, sx }) => {
+  return (
+    <Box sx={{ ...sx }}>
+      <Typography sx={{ fontWeight: 800, fontSize: '0.8rem' }}>Egg Groups:</Typography>
+      <Container>
+        <Box display="flex" alignItems="center">
+          <img src="/img/pm0000_00_00_00_L.webp" alt="Pokemon Egg" width="16px" height="20px" />
```

Comment:
We should acquire egg group icons, up to you if you want to make that a separate ticket or add it to this one.

Reply:
will be done in a different ticket